trace-java-client/src/main/java/com/amazon/opentelemetry/trace/client/JaegerTraceEmitClient.java

```diff
@@ -89,13 +91,18 @@ private Tracer getTracer(Service service) {
   }
 
   private static Tracer createJaegerTracer(String collectorUrl, Service svc, int flushIntervalMillis) {
-    HttpSender sender = new HttpSender.Builder(collectorUrl + "/api/traces").build();
+    HttpSender sender = null;
+    try {
+      sender = new HttpSender.Builder(collectorUrl + "/api/traces").build();
+    } catch (TTransportException e) {
+      throw new RuntimeException("Can't create http sender");
```

Comment:
Should we log any information about `e` here as well?

Reply:
Added